microk8s-resources/actions/enable.openebs.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source "$SNAP/actions/common/utils.sh"
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+OPENEBS_NS="openebs"
+
+# Check if iscsid is installed 
+if ! systemctl is-enabled iscsid | grep enabled &> /dev/null
+  then
+    echo "iscsid is not available or enabled.  Make sure iscsi is installed on all nodes."
+    echo "To enable iscsid: "
+    echo "      sudo systemctl enable iscsid"
+    echo "Please refer to the OpenEBS prerequisites (https://docs.openebs.io/docs/next/prerequisites.html)"
+    exit   
+fi
+
+"$SNAP/microk8s-enable.wrapper" dns
+"$SNAP/microk8s-enable.wrapper" helm3
+
+
+# make sure the "openebs" namespace exist
+$KUBECTL create namespace "$OPENEBS_NS" > /dev/null 2>&1 || true
+
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
+
+$HELM repo add openebs https://openebs.github.io/charts
+$HELM repo update
+$HELM -n openebs install openebs openebs/openebs \
+    --set varDirectoryPath.baseDir="$SNAP_COMMON/var/openebs/" \
+    --set jiva.defaultStoragePath="$SNAP_COMMON/var/openebs/"
```

Comment:
I noticed the ndm sparse path wasn't set to inside the snap so I looked through the doc I think this is all the /var/openebs directories that need setting. I added them and removed trailing slashes from the existing two.
reference:
https://github.com/openebs/charts/blob/master/charts/openebs/README.md

```suggestion
    --set varDirectoryPath.baseDir="$SNAP_COMMON/var/openebs" \
    --set jiva.defaultStoragePath="$SNAP_COMMON/var/openebs" \
    --set localprovisioner.basePath="$SNAP_COMMON/var/openebs/local" \
    --set ndm.sparse.path="$SNAP_COMMON/var/openebs/sparse"
```

Reply:
Thanks for the review.  Good catch.  I will update the PR. 👍